sirepo/simulation_db.py

```diff
@@ -1241,37 +1248,51 @@ def _validate_fields(data):
             _validate_enum(val, sch_field_info, sch_enums)
             _validate_number(val, sch_field_info)
 
+
 def _validate_name(data):
     """Validate and if necessary uniquify name
 
     Args:
         data (dict): what to validate
     """
-    starts_with = pkcollections.Dict()
     s = data.models.simulation
+    sim_type = data.simulationType
+    sim_id = s.simulationId
     n = s.name
+    f = s.folder
+    c = None
+    m = re.search(r'(.+) ([0-9]+)$', n)
+    if m:
+        n = m.group(1)
+        c = int(m.group(2))
+    if c is not None:
+        _assert_sim_copy_num(s.name, c)
```

Comment:
I don't think this is correct. The user should be able to name a sim anything they want. 

The goal was to eliminate a (potential) bug when the sim copy loop got to maxSimCopies, which it should never do.